source/_posts/angular-router-guard-rbac.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Angular Guard for Role-Based Access Control (RBAC) 
+author: A M Sanjeev
+avatar: a-m-sanjeev.jpg
+description: Learn about Angular Guard, AuthService, AuthGuard Implementation, and Routing Module Implementations and create a simple Angular app with user and admin roles and role-based authorization.
+date: 2022-09-11
+tags: [Angular, Security]
+id: 
+relatedPost:
+---
+{% image_fw 1.78 banner.png "Angular Guard for Role-Based Access Control (RBAC)" %}
+
+Angular Guard for Role-Based Access Control (RBAC)
+==================================================
+
+We learned how to use the [Web Security Academy](https://websecurity-academy.com/?utm_source=blog&utm_medium=link&utm_campaign=angular-otp-post&_ga=2.165189500.2131069689.1662582129-1733203209.1662582129) training program [budget-angular](https://github.com/bartosz-io/budget-angular) to request and verify one-time passwords in the preceding post [**Angular OTP (one-time password) Verification 🔢**](https://dev-academy.com/angular-otp-verification/). We talked about two-factor authentication (2FA), time-based one-time passwords, and using Angular to request OTP verification.

```

Comment:
Change **utm_campaign** to this post id (angular-router-guard-rbac) and remove **_ga**